<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DatabaseConnectionWidget</class>
 <widget class="QWidget" name="DatabaseConnectionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="documentMode">
      <bool>false</bool>
     </property>
     <widget class="ExplorerWidget" name="explorerTab">
      <attribute name="icon">
       <iconset theme="document-open">
        <normaloff>.</normaloff>.</iconset>
      </attribute>
      <attribute name="title">
       <string>Explorer</string>
      </attribute>
     </widget>
     <widget class="QueryWidget" name="queryTab">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <attribute name="icon">
       <iconset theme="document-edit">
        <normaloff>.</normaloff>.</iconset>
      </attribute>
      <attribute name="title">
       <string>Query</string>
      </attribute>
     </widget>
     <widget class="SqlWidget" name="sqlTab">
      <attribute name="icon">
       <iconset theme="document-import">
        <normaloff>.</normaloff>.</iconset>
      </attribute>
      <attribute name="title">
       <string>SQL</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QueryWidget</class>
   <extends>QWidget</extends>
   <header>QueryWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SqlWidget</class>
   <extends>QWidget</extends>
   <header>SqlWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ExplorerWidget</class>
   <extends>QWidget</extends>
   <header>ExplorerWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
